transports/quic/src/config.rs

```diff
@@ -59,6 +59,7 @@ pub struct Config {
     /// If support for draft-29 is enabled servers support draft-29 and version 1 on all
     /// QUIC listening addresses.
     /// As client the version is chosen based on the remote's address.
+    #[deprecated(note = "QUIC draft versions are no longer supported")]
     pub support_draft_29: bool,
```

Comment:
we should probably add an allow on our usage of this field, in `transports/quic/src/transport.rs:91`